docs/getting-started/configuration/crossplane.yaml

```diff
@@ -3,6 +3,16 @@ kind: Configuration
 metadata:
   name: getting-started-with-aws
   annotations:
+    meta.crossplane.io/maintainer: Upbound <support@upbound.io>
+    meta.crossplane.io/source: github.com/upbound/universal-crossplane
+    meta.crossplane.io/license: Apache-2.0
+    meta.crossplane.io/description: |
+      An introductory example to UXP and Composition for AWS.
+    meta.crossplane.io/readme: |
+      An introductory example to UXP and Composition for AWS.
+      This will enable provisioning of an RDS database instance.
+
+      [Quickstart](https://docs.upbound.io/quickstart/community-providers/)
```

Comment:
@jastang looks like this quickstart involves installing a provider directly -- is that what we want to link to for this package?

Reply:
ah, good catch! This was just a placeholder. Looks like the original guide hasn't been moved to the new docs site yet. I'll follow up here

Reply:
The appropriate docs are only available in the archive for now according to @plumbis, so I'll remove this until they're available in docs.upbound.io